app/controllers/v0/profile/email_addresses_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module V0
+  module Profile
+    class EmailAddressesController < ApplicationController
+      before_action { authorize :vet360, :access? }
+
+      def create
+        email_address = Vet360::Models::Email.with_defaults(@current_user, email_address_params)
+
+        if email_address.valid?
+          response    = service.post_email email_address
+          transaction = AsyncTransaction::Vet360::EmailTransaction.start(@current_user, response)
+
+          render json: transaction, serializer: AsyncTransaction::BaseSerializer
+        else
+          raise Common::Exceptions::ValidationErrors, email_address
+        end
+      end
+
+      def update
+        email_address = Vet360::Models::Email.with_defaults(@current_user, email_address_params)
+
+        if email_address.valid?
+          response    = service.put_email email_address
+          transaction = AsyncTransaction::Vet360::EmailTransaction.start @current_user, response
```

Comment:
What does `.start` do here? Is that an ActiveRecord method?

Reply:
@emersonthis - it is a new class method on the base class:

https://github.com/department-of-veterans-affairs/vets-api/blob/4bf4da304c0602c802768a69eae930a5b2a618ce/app/models/async_transaction/vet360/base.rb#L6-L15